src/charm.py

```diff
@@ -187,22 +255,21 @@ def configure_workload_container(self, event) -> None:
                 logger.info(msg)
                 self.unit.status = MaintenanceStatus(msg)
                 container.add_layer(CONTAINER_NAME, pebble_config, combine=True)
+                container.pebble.replan_services()
+        else:
+            self.unit.status = WaitingStatus("waiting for Pebble to start")
```

Comment:
This should be "Waiting" rather than "waiting" per the new guidelines on sentence casing for status messages, but I think we should also mention which container we're waiting for pebble to start in.